client/src/components/AnswerQuestionBar.jsx

```diff
@@ -36,8 +36,35 @@ overflow: hidden;
 resize: none;
 `
 const SubmitAButton = styled.button `
+background-color: rgb(0, 0, 0);
+border-color: rgb(0, 0, 0);
+border-radius: 3px;
+border-width: 1px;
+color: rgb(255, 255, 255);
+display: inline-block;
+padding-bottom: 8px;
+padding-top: 8px;
+padding-right: 16px;
+padding-left: 16px;
+text-align: center;
+height: 36px;
+width: 77px;
+margin-right: 16px;
```

Comment:
You can shorten up the code with this syntax: "padding: 8px 16px" the first spot indicates top/bottom and the second represents left/right. That way you only need to write padding once.